commands/test/test-command.ts

```diff
@@ -153,22 +153,62 @@ export const sendQuestion = async (interaction: any) => {
         console.log(userResponses);
         console.log(interaction.user.id);
 
-
-
         const bestMatch = await findMatchingUser(interaction.user.id, userResponses);
-
+       
         if (bestMatch) {
-            interaction.user.send(`Dein bester Gesprächspartner ist: **@${bestMatch.username}**.`);
-            interaction.user.send("Als nächstes schreibst du deinem Partner, indem du auf seinen Namen auf dem Contraversum-Server klickst 👆 und ihm eine Nachricht sendest.");
-            interaction.user.send("Dies sind drei Fragen bei denen ihr euch unterscheidet:");
-
-            // Send the best match that they have been matched with the user
-            const bestMatchUser = await client.users.fetch(bestMatch.userId);
-            if (bestMatchUser) {
-                bestMatchUser.send(`Hey 👋, du wurdest mit: **@${interaction.user.username}** gematched.`);
-                bestMatchUser.send("Ihr habt euch bei diesen drei Fragen beispielsweise unterschieden:");
+            const guildId = process.env.GUILD_ID;
+            if (!guildId) {
+                console.error('GUILD_ID is not found');
+                return;
+            }
+            const guild: Guild | undefined = client.guilds.cache.get(guildId);
+            if (!guild) {
+                console.error('Guild not found');
+                return;
+            }
+
+            const member = guild.members.cache.get(interaction.user.id);
+            if (!member) {
+                console.error('Member not found');
+                return;
             }
-            conversationStarter(interaction, bestMatch.userVector, userResponses, bestMatchUser)
+
+            bestMatch.memberId = await guild.members.fetch(bestMatch.userId);
+            if (!bestMatch.memberId) {
```

Comment:
This would just break if the user is still in our DB but not on the server anymore meaning the user would be left with no match and would not know what to do. Therefore we must change the findMatchingUser function to check instead of here. The logic should be like this:

1. find matching user
2. check if machting user is member
2.1 if not delete user from db and run again
3. Finish

Would also be a cool exercise to use and understand recursion